src/Riok.Mapperly/Helpers/SymbolExtensions.cs

```diff
@@ -103,6 +103,29 @@ internal static bool HasImplicitInterfaceMethod(this ITypeSymbol symbol, INamedT
         return !methodInterImplementaton.ExplicitInterfaceImplementations.Any();
     }
 
+    internal static bool HasImplicitInterfaceProperty(this ITypeSymbol symbol, INamedTypeSymbol inter, string propertyName)
```

Comment:
Do we need the limitation of properties here?
I'd probably prefer the approach of https://github.com/riok/mapperly/pull/341#discussion_r1167471503.